### DISCUSSION TITLE
k8s 1.28 compatibility

### DISCUSSION THREAD

**Discussion by ivisilva-ap** [argoproj/argo-workflows#12695] [Q&A] [answered]
Hi folks,

Are there any plans to test Argo Workflows using Kubernetes 1.28?

I saw here https://argo-workflows.readthedocs.io/en/latest/releases/#kubernetes-compatibility-matrix that the compatibility is until Kubernetes 1.27.

Thanks
Ivi

**Comment by agilgur5** [comment, marked as answer]
It is now tested on k8s 1.29 after #12854
